modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/modals/BatchActionModal.tsx

```diff
@@ -99,11 +99,9 @@ const BatchActionModal: React.FC<IBatchActionModalProps> = ({
 						trigger={
 							<ClayButton
 								className='button-root'
-								displayType='unstyled'
+								displayType='secondary'
 							>
-								<span className='text-truncate'>
-									{selectedKey}
-								</span>
+								<span>{selectedKey}</span>
```

Comment:
Hey @fmarcio why did you remove `text-truncate` classname?

Reply:
Hey @interaminense I removed it because this class was avoiding the icon in the button to be displayed correctly
Btn with `text-truncate` class:
![change-permission-btn](https://github.com/liferay-ac/liferay-portal/assets/38669344/47345520-082c-470e-bcaa-594b97975d38)